autogpts/autogpt/autogpt/app/main.py

```diff
@@ -116,12 +122,6 @@ async def run_auto_gpt(
         skip_news=skip_news,
     )
 
-    # Set up logging module
-    configure_logging(
-        **config.logging.dict(),
-        tts_config=config.tts_config,
-    )
-
```

Comment:
This change broke `--debug`. `apply_overrides_to_config` propagates the `--debug` flag to the configuration, so `configure_logging` has to be after `apply_overrides_to_config`.

Reply:
Fixed in cf00c33f907b12e48218404f849c11beb568001b